app/policies/user_policy.rb

```diff
@@ -3,6 +3,19 @@ def index?
     user.internal?
   end
 
+  def create?
+    user.permission?(:manage_user)
+  end
+
+  def update_security_roles?
```

Comment:
this new policy action controls the controller and the user form for this scenario